### ISSUE TITLE
Establish Process for Mentee and Mentor Selection

### ISSUE THREAD

**Issue by naokomc** [WordPress/wp-contributor-working-group-tracker#9]
Establish clear criteria for selecting mentees and mentors from the applications. These criteria will ensure that the pairing process is transparent, fair, and inclusive so that the program aligns with the community's diversity and inclusion principles.

Below are proposed criteria.

---
### Required
- Has a WordPress.org account
- Has a Make WordPress Slack account
- No unresolved CoC violation
- No unresolved [WordPress Trademark Policy](https://wordpressfoundation.org/trademark-policy/) or [100% GPL rule](https://make.wordpress.org/community/handbook/wordcamp-organizer/first-steps/helpful-documents-and-templates/100-gpl-vetting-checklist/) violation
- Has at least one activity on their profile

### Bonus
- Expressed clear learning goal
- Has experience in other OSS projects
- Account age (wide variety across the cohort, if possible)

---
## 2. Mentor Selection Criteria

- Has an active WordPress.org account (last login is within a month)
- Has a Make WordPress Slack account
- Has contribution experience in the relevant team
- No unresolved CoC violation
- No unresolved [WordPress Trademark Policy](https://wordpressfoundation.org/trademark-policy/) or [100% GPL rule](https://make.wordpress.org/community/handbook/wordcamp-organizer/first-steps/helpful-documents-and-templates/100-gpl-vetting-checklist/) violation
- Has consistently shown effective communication skills

### Bonus

- Has previous mentorship experience
- Has a project proposal
- Account age (wide variety across the cohort, if possible)

---
## 3. Matching Criteria

- ~~Team interest and experience~~ Team interest & experience level compatibility
- Skill level compatibility
- Goal & support compatibility
- Time zone compatibility
- [Optional] No or little previous interaction (to encourage new connections)
- [Optional] Language proficiency

---
## 4. Vetting & Selection

The members of the Contributor Working Group will vet the applications and make the final selections. 

---
Please leave a comment if you have feedback or ideas for improving these. Your input help us shape an inclusive and impactful mentorship program! 😃

**Comment by moranjosep** [comment]
[Coments_Josep-Moran.md](https://github.com/WordPress/wp-contributor-working-group-tracker/files/13751004/Coments_Josep-Moran.md)

[Edit by @naokomc] For better readability, I pasted the comment in the .md file below.

---
# comments

## Mentee Selection Criteria

 **Proposals**

**They might have no activity on their profile yet be very active and beneficial in the community.**

I know of a specific case where, despite making few visible contributions, they are a very active part as a volunteer at WordCamps in Spain. This is an activity that cannot be quantified through their profile on WordPress.org.
It's an option worth considering since WordCamps are where I've been asked the most about what the mentoring course is and how it functions.

**Bonus: expressed a clear learning objective.**

**Proposal**

Expressing a clear learning objective might not be enough; they should express their willingness to collaborate continuously based on their possibilities over time once the training is completed.
In my humble opinion, the strength of this training lies in creating new, stable, and consistent collaborators, not just educating them.

## Matching Criteria

**Team interest and experience**

**Proposal**

Someone with a strong willingness to collaborate might lack experience in a team or knowledge of its functioning. I believe it's better to assess this point from the perspective of interest rather than experience.

In this case, I speak for myself. I had translated 10 strings in 100 days. During the mentoring, it was 1,200 strings. Afterward, it became 10,000 in 90 days. I think that's the real goal of mentorships. This is an example of someone with high interest and little initial knowledge.

I hope that with these comments, I can contribute something positive to this fascinating project.
Best regards.
Josep Morán.

**Comment by tobifjellner** [comment]
Any thoughts around "account age"? We may want to create a good mix of mentors with long/wide experience from the project and mentors that joined more recently (remember that "mentoring is also a learning experience").
And Mentees may be relatively recent contributors, or someone may have been around a long time, but wishes to widen their understanding of the project and join new areas.

**Comment by naokomc** [comment]
Thanks for the feedback @moranjosep !

> They might have no activity on their profile yet be very active and beneficial in the community.

With your feedback and the minor modification to the announcement post about the past contribution, I will move "Has at least one activity on their profile" from the Required to the Bonus section. You can see the exchange between Olga/Hari/me from [here](https://docs.google.com/document/d/1UNQgnxnDOk-kJYiUAmGEUwMdmL0Q87xAHvSJn4gozgk/edit?disco=AAAA9zeK2SA) if interested.

> Team interest and experience

We are on the same page, but I agree I needed clarification. I meant the mentee's interest should match the mentor's experience. Also, a brand-new contributor can be matched with a moderately experienced mentor, while relatively experienced mentees can be matched with a more advanced mentor. I wanted to let you know my intention for this item; I hope this is clear.

I think it is better to phrase "Team interest & experience level compatibility" – any other improvement suggestion is welcome.

@tobifjellner 
Account age is an excellent additional consideration for the diversity of the cohort. I will add it to the Bonus section of the selection criteria.

**Comment by naokomc** [comment]
I didn't mention who will be vetting and selecting the cohort and how.
I propose the following:

The members of the Contributor Working Group will vet the applications and make the final selections. 

(added to the issue description)

**Comment by angelasjin** [comment]
Can we please check for compliance with the Trademark and GPL for mentors and mentees? These are both important to the WordPress of today and tomorrow, so it is of particular relevance for this work.

**Comment by naokomc** [comment]
Thanks @angelasjin – added this to both.

- No unresolved [WordPress Trademark Policy](https://wordpressfoundation.org/trademark-policy/) or [100% GPL rule](https://make.wordpress.org/community/handbook/wordcamp-organizer/first-steps/helpful-documents-and-templates/100-gpl-vetting-checklist/) violation

**Comment by naokomc** [comment]
For Cohort 2, we finished the selection process using these criteria.
In addition:

1. We used a Google Spreadsheet to vet all applicants (example of mentor vetting) ![Image](https://github.com/WordPress/wp-contributor-working-group-tracker/assets/34067/318e9278-51c7-4baa-a17a-8141c2f50530)
2. Based on the total score & overall application, we selected mentors first
3. Then, we assigned mentees based on their interest and experience ![Image](https://github.com/WordPress/wp-contributor-working-group-tracker/assets/34067/add14280-2e1a-4826-a83b-3a3c8a283a95)
4. We contacted mentors to confirm the assigned mentees and made some adjustments.
